lib/rspec/matchers/built_in/count_expectation.rb

```diff
@@ -56,14 +56,8 @@ def times
 
       private
 
-        if RUBY_VERSION.to_f > 1.8
-          def cover?(count, number)
-            count.cover?(number)
-          end
-        else
-          def cover?(count, number)
-            number >= count.first && number <= count.last
-          end
+        def cover?(count, number)
+          count.cover?(number)
```

Comment:
Any reason why this wasn't refactored to use it directly?

Reply:
Nice catch. Let me inline this quickly.